mix.exs

```diff
@@ -82,7 +85,7 @@ defmodule EctoShorts.MixProject do
 
         "Misc Modules": [
           EctoShorts.Actions.Error,
-          EctoShorts.Repo
+          EctoShorts.Support.Repo
```

Comment:
Maybe we should @moduledoc false this

Reply:
@moduledoc false is in the module. I removed it from the list.